public/app/plugins/datasource/grafana-pyroscope-datasource/README.md

```diff
@@ -0,0 +1,28 @@
+# Grafana Pyroscope Data Source - Native Plugin
+
+Grafana ships with **built in** support for Grafana Pyroscope, an open source continuous profiling platform.
+
+## What is Grafana Pyroscope?
+
+Grafana Pyroscope is an open source continuous profiling platform. It will help you:
+* Find performance issues and bottlenecks in your code
+* Use high-cardinality tags/labels to analyze your application
+* Resolve issues with high CPU utilization
+* Track down memory leaks
+* Understand the call tree of your application
+* Auto-instrument your code to link profiling data to traces
+
+## [Live Demo](https://demo.pyroscope.io/)
```

Comment:
Do we want to link to the demo here?

Reply:
I wish we had a new demo, but we don't have that right now, I think I would remove it.

Reply:
I'd maybe add a link to here: https://grafana.com/docs/grafana/latest/datasources/grafana-pyroscope/